REQUIRE

```diff
@@ -1,3 +1 @@
-julia 0.3-
-Dates
-
+julia 0.4-
```

Comment:
If you'd like it to install it right now, Julia 0.4- is correct ("Julia 0.4 prerelease"). `julia 0.4` means `>=0.4.0`. Not really the clearest thing in the world, is it :D